### PR TITLE
Added NetworkTransferBytes and NetworkReceiveBytes to Allocation

### PR DESCRIPTION
**Pull request by mbolt35**
CAdvisor Queries for tracking Pod Total Transferred/Received Bytes

### CODE REVIEW

**Comment by nikovacevic** [comment]
Any notes on how this was tested?

File: pkg/kubecost/allocation.go [hunk 1]
```diff
@@ -63,6 +63,8 @@ type Allocation struct {
 	GPUHours                   float64               `json:"gpuHours"`
 	GPUCost                    float64               `json:"gpuCost"`
 	GPUCostAdjustment          float64               `json:"gpuCostAdjustment"`
+	NetworkTransferBytes       float64               `json:"networkTransferBytes"`
+	NetworkReceiveBytes        float64               `json:"networkReceiveBytes"`
```
Comment (nikovacevic) [inline pkg/kubecost/allocation.go:67]: Should we also add these to the `MarshalJSON`, or do you want to keep them out of the JSON payload for now?

**Review by nikovacevic — COMMENTED** [review]
Code looks great! Just the two questions about testing and JSON.

File: pkg/kubecost/allocation.go (hunk 1 above)
Comment (mbolt35) [reply pkg/kubecost/allocation.go:67]: Great Googly Moogly

Comment (mbolt35) [reply pkg/kubecost/allocation.go:67]: > Any notes on how this was tested?

I had logging after the query that seemed to indicate the data coming through was correct, but after adding JSON encoding, I'm seeing 0s for both on the allocation request for 7d, so it seems I need to find a better way to end-to-end test.

Suggestions other than observing on the JSON result?

Comment (mbolt35) [reply pkg/kubecost/allocation.go:67]: Added that, the fields to the equality check, and also to `add()`

**Comment by mbolt35** [comment]
image

**Review by nikovacevic — APPROVED** [review]

**Pull request merged**